src/main/java/pl/touk/sputnik/configuration/GeneralOption.java

```diff
@@ -18,15 +18,15 @@
     MESSAGE_PROBLEM_FORMAT("message.problemFormat", "Sputnik problem format. {0}: reporter, {1}: message", "There is a problem with {0}: {1}"),
     MESSAGE_SCORE_PASSING_COMMENT("message.scorePassingComment", "Comment when no errors are found", "Perfect!"),
 
-    CONNECTOR_TYPE("connector.type", "Connector: <stash|gerrit>", ConnectorType.GERRIT.getName()),
+    CONNECTOR_TYPE("connector.type", "Connector: <stash|gerrit|github>", ConnectorType.GERRIT.getName()),
     HOST("connector.host", "Connector server host", "localhost"),
     PORT("connector.port", "Connector server port", "80"),
     PATH("connector.path", "Connector server path", ""),
     USE_HTTPS("connector.useHttps", "Connector use https?", "false"),
     USERNAME("connector.username", "Connector server username", "user"),
     PASSWORD("connector.password", "Connector server password", "password"),
-    PROJECT_KEY("connector.projectKey", "Connector server projectKey", null),
-    REPOSITORY_SLUG("connector.repositorySlug", "Connector server repositorySlug", null),
+    OWNER("connector.owner", "Connector server owner", null),
```

Comment:
I know it was some time ago and this PR is merged, but this change will break clients who have `connector.repositorySlug` in their configuration files. 
Moreover, not all elements have been changed: https://github.com/TouK/sputnik/search?utf8=%E2%9C%93&q=%22connector.repositorySlug%22+%22connector.projectKey%22&type=Code